### DISCUSSION TITLE
Runtime colors / theme properties

### DISCUSSION THREAD

**Discussion by its-monotype** [formaat-design/reshaped#191] [Q&A] [answered]
Is it possible to get theme color as a JS variable, for example, to apply it to a chart?

**Comment by blvdmitry** [comment, marked as answer]
Currently js definitions are not shipped to not double the theme size in the css/js bundles. 

However, you can still use css variables right in your js string:
"var(--rs-color-background-primary)"

As an example, we use it this way on our website to style the Stripe payment form. Let me know if this works for your case

**Reply by its-monotype** [reply]
Ah, I hadn't thought of it, that makes sense. Works great, here's an example: https://codesandbox.io/s/nostalgic-haze-76dxwj.

**Reply by blvdmitry** [reply]
Looks amazing! 🔥
